data/json/recipes/recipe_others.json

```diff
@@ -848,7 +848,10 @@
     "difficulty": 5,
     "time": "2 h",
     "autolearn": true,
-    "proficiencies": [ { "proficiency": "prof_knapping", "required": true } ],
+    "proficiencies": [
+      { "proficiency": "prof_knapping", "required": false, "time_multiplier": 2, "fail_multiplier": 10 },
```

Comment:
This may or may not actually need a different proficiency than knapping.

Stone axe is a ground stone tool; this means that you don't smash rocks together to make a stone axe head; you grind down a suitable rock to necessary shape using sand, water and a larger rock (represented by ANVIL 1 quality), then make a hole in a stick that you'd use as a haft (in this recipe the most likely tools for this are stone chisel, which is a ground stone tool too, and a rock to strike it), fire-harden the wood around the hole, and hammer the stone axe head into the hole so that it holds well.

Reply:
Sounds like it would be worth considering a Stoneworking proficiency and having two versions of this recipe. I can find versions made with knapping as well.